uqcsbot/__main__.py

```diff
@@ -55,6 +55,7 @@ async def main():
         "jobs_bulletin",
         "latex",
         "member_counter",
+        "morse",
```

Comment:
This is a very small nitpick, but could this be moved a line lower to keep it in alphabetical order.